src/widgets/shared-utils.tsx

```diff
@@ -0,0 +1,14 @@
+import { newWorkspaceItem } from "@openmrs/esm-api";
+
+export const openWorkspaceTab = (componentToAdd, componentName, params?) => {
```

Comment:
Same again - love the simplifcation and everything. Just fix the other branch, update this one and we can merge.

Reply:
Yeah, let me close this one since the changes have already been merged in another PR.